exercises/allergies/uAllergyTests.pas

```diff
@@ -13,39 +13,39 @@   AllergyTests = class(TObject)
     procedure No_allergies_means_not_allergic;
 
```

Comment:
It will help when we get the generators going.  First test always ignored, but the pattern is there.  Makes templating easier.